modules/manage/partials/tiered-storage.adoc

```diff
@@ -1063,7 +1063,15 @@ rpk topic describe-storage <topic_name> --print-all
 
 See the xref:reference:rpk/rpk-topic/rpk-topic-describe-storage.adoc[reference] for a list of flags you can use to filter the command output.
 
-TIP: A constant stream of data is necessary to build up the log segments and roll them into object storage. To see new log segments faster, you can edit the `segment.bytes` topic-level property. Or, you can edit the xref:reference:properties/object-storage-properties.adoc#cloud_storage_segment_max_upload_interval_sec[`cloud_storage_segment_max_upload_interval_sec`] property, which sets the frequency with which new segments are generated in Tiered Storage, even if the local segment has not exceeded the `segment.bytes` size or the `segment.ms` age.
+[TIP]
+====
+A constant stream of data is necessary to build up the log segments and roll them into object storage. This upload process is asynchronous. You can monitor its status with the following metrics:
+
+* xref:reference:public-metrics-reference.adoc#redpanda_cloud_client_uploads[`/public_metrics/redpanda_cloud_client_uploads`]
```

Comment:
I'm not sure what the intent is here but this metric (redpanda_cloud_client_uploads) is not directly relevant for monitoring status/progress of uploads. It monitors the number of PUT requests we do to S3 but these might be or might NOT be related to tiered storage.

The `vectorized_ntp_archiver_pending` is ok.